schematics/ng-add/setup-project/register-locale.ts

```diff
@@ -22,7 +22,7 @@ export function registerLocale(options: Schema): Rule {
     const appModulePath = getAppModulePath(host, getProjectMainFile(project));
     const moduleSource = getSourceFile(host, appModulePath);
 
-    const locale = options.locale || 'en_US';
+    const locale = options.locale || options.i18n || 'en_US';
```

Comment:
I suggest you could start to deprecate `i18n` options since v7.0.0 and remove it completely on v8.0.0. Basically we are not subject to breaking change but breaking changes should be made cautiously and migration phrase should be offered.

Personally I am glad that `--i18n` is replaced by `--locale` --- it is more accurate then the previous term.